golang 1.9 and plugins updates.

jcvenegas opened the pull request:
Update golang to version 1.9.2 to avoid issues building   third party modules.

Update configuration from plugins.

jcvenegas committed:
config: Updte golang to 1.9.2

Install golang 1.9 instead of 1.8

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

jcvenegas committed:
plugins: Update Plugins

Update plugins

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

chavafg commented on jobs/virtcontainers-fedora-26-PR/config.xml:
```diff
@@ -92,6 +92,8 @@ set -e
 
 export ghprbPullId
 export ghprbTargetBranch
+export ghprbAuthorRepoGitUrl
```
Can you add a comment in the commit message that these changes were to comply with the virtcontainers CI changes? 
https://github.com/containers/virtcontainers/commit/1a638c0f173b9aa06d8e03943ae604655f1719b8
Or better, add a new commit.

jcvenegas replied:
@chavafg , added new commit !

sboeuf commented:
@jcvenegas Once you've addressed @chavafg's comment, this LGTM

jcvenegas committed:
config: comply with the virtcontainers CI changes

Needed by:

https://github.com/containers/virtcontainers/commit/1a638c0f173b9aa06d8e03943ae604655f1719b8

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

jcvenegas commented:
@chavafg ping

chavafg commented:
lgtm
